inductiva/templates/coastal_area/swash/input.sws.jinja

```diff
@@ -0,0 +1,29 @@
+$$$ HEADING
+PROJECT 'inductiva' '00'
+$$$ PHYSICAL CONSTANTS
+SET temp = 14.0
+SET salinity = 0.1
+SET dynvis = 0.001
+MODE DYNAMIC TWODIMENSIONAL
+CGRID 0. 0. 0. 1200.000000 400.000000 299 99
+INPGRID BOTTOM REGULAR 0.0 0.0 0. 299 99 4.000000 4.000000
+READINP BOTTOM 1. 'bathymetry.bot' 5 0 FREE
```

Comment:
Presumably, to add the bathymetry is just to change here.

Reply:
Not exactly. The region defined by the grid in the bathymetry file is not necessarily that to be simulated, nor is its resolution. It is more involved than that, but we have ideas that we worked on in a previous project.

Reply:
I would say that the grid and the resolution can also be responsibility of the user. But I will leave to you, how you prefer doing it.